lib/shoulda/matchers/active_model/validator_with_captured_range_error.rb

```diff
@@ -0,0 +1,11 @@
+module Shoulda
+  module Matchers
+    module ActiveModel
+      module ValidatorWithCapturedRangeError
+        def messages_description
+          ' RangeError: ' + captured_range_error.message.inspect
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.